src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.test.js

```diff
@@ -1,10 +1,107 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
 
-// import ImageViewer from 'index';
+import AttachmentViewer from './index';
```

Comment:
Linter should complain about this; should be `import AttachmentViewer from './';`

Reply:
dat zit overal. ik vind dat geen issue. als je die wil: voeg maar een regel toe aan de linter